### ISSUE TITLE
Precompiled binary releases

### ISSUE THREAD

**Issue by bdewater** [PJK/libcbor-ruby#9]
From the discussion over at https://github.com/cedarcode/webauthn-ruby/pull/88#issuecomment-507040249 

> It breaks the "just run bundle install and you're done" workflow we have now. It would be nice if we can package the library with the gem or provide pre-installed binaries like https://github.com/cowboyd/libv8

**Comment by PJK** [comment]
I fully agree that this would be the ideal end state. As an intermediate step, we could at least compile from source if a system version is not available (libcbor builds are rather fast).
